tensorflow/cc/framework/gradient_checker.cc

```diff
@@ -47,7 +47,7 @@ Status ComputeTheoreticalJacobianTranspose(
   auto dy_data_flat = dy_data.flat<T>();
   dy_data_flat.setZero();
 
-  // Compute the theoretical Jacobian one row at a time by backproping '1.0'
+  // Compute the theoretical Jacobian one row at a time by back probing '1.0'
```

Comment:
I believe this is meant to be "backproping" as in "back propagating" and not "back probing"